client/src/app/shared/loading-bar/loading-bar.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+@Injectable()
+export class LoadingBarService {
```

Comment:
Let's rename this to `ProgressBarService`.

Reply:
Shall I rename the component to `ProgressBarComponent`?